src/provider/view/components/CircleButton/CircleButton.tsx

```diff
@@ -0,0 +1,49 @@
+import * as React from 'react';
+
+import './CircleButton.scss';
+
+enum IconType {
+    accept = 'accept',
+    cancel = 'cancel',
+    close = 'close',
+    hide = 'hide',
+    pin = 'pin',
+    dismiss = 'dismiss'
```

Comment:
Can we make the keys CONST_CASE? (`ACCEPT`, `CANCEL`, etc). Actual key values can be uppercase, lowercase, or numeric.

Should also export, so the enum can be used to build the props.